src/components/Header/Header.jsx

```diff
@@ -12,6 +13,9 @@ const Header = () => {
         at the top left and the "Profile" link at the top right. The logo
         has already been imported for you (see line 2). The completed header
         should route the user to the Profile page correctly. */}
+      <img id="logo" src={toody}></img>
```

Comment:
[nit] if our closing tag immediately follows our opening tag we should use a self closing tag, e.g. `<img .... .... />` so that we don't need two tags to convey the same